package.json

```diff
@@ -29,7 +29,7 @@
     "serve": "npm-run-all 'serve-examples {@}' --",
     "serve-examples": "./bin/serve $(if [ -z \"$*\" ]; then echo \"examples\"; fi)",
     "serve-live": "./bin/serve",
-    "start-server": "node ./server.js --root build/ --host 0.0.0.0 --port ${PORT:-80} $(if [ -n \"${BOT_PROXY:-}\" ]; then echo \"--bot-proxy ${BOT_PROXY}\"; fi)",
+    "start-server": "node ./server.js --root build/ --host 0.0.0.0 --port ${PORT:-80} $(if [ -z \"${BOT_PROXY:-}\" ]; then echo \"--bot-proxy ${BOT_PROXY}\"; fi)",
```

Comment:
`-z` means "is empty". So what you're doing here is saying, if and only if `$BOT_PROXY` *doesn't* exist, should we add the `--bot-proxy` argument. Which doesn't make any sense. Unless I've misunderstood something.

Reply:
well here it didn't work for me by running with -n it was still passing the argunment

Reply:
@tbille so I would suggest that probably means that the `--bot-proxy` argument is fundamentally broken, or maybe the value of the `${BOT_PROXY}` variable is wrong. But the fix isn't to simply reverse the conditional to something that makes no sense. The fix *may* be to remove `--bot-proxy` altogether, but only if you understand what you're doing.

Reply:
ok when the var is defined but empty it still works

Reply:
```
$ BOT_PROXY=""
$ if [ -n ${BOT_PROXY:-} ]; then echo \"--bot-proxy ${BOT_PROXY}\"; fi
--bot-proxy
```

Reply:
You need to wrap `${BOT_PROXY:-}` in quotes (`"${BOT_PROXY:-}"`) or it won't behave as you expect:

``` bash
$ [ -n  ] && echo true || echo false
true
```